cdk/README.md

```diff
@@ -0,0 +1,49 @@
+Pre-requisites
+1) Create secret in Secrets Manager as per https://docs.aws.amazon.com/AmazonECS/latest/developerguide/linux-gmsa.html#linux-gmsa-setup
```

Comment:
Please specify the corresponding secret to create used in the Test environment in `cdk/data.json`

Reply:
Added in latest rev